### ISSUE TITLE
help users without internet access with go plugin

### ISSUE THREAD

**Issue by ktsaou** [netdata/netdata#5475]
When there is no internet access, go.d.plugin cannot be downloaded and the installer fails with this:

```

 --- Install go.d.plugin --- 
[/usr/src/netdata.git]# curl -L --connect-timeout 10 --retry 3 https://github.com/netdata/go.d.plugin/releases/download/v0.1.0/go.d.plugin-v0.1.0.linux-amd64 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (77) error setting certificate verify locations:
  CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: none
 FAILED   

 ABORTED  Cannot download https://github.com/netdata/go.d.plugin/releases/download/v0.1.0/go.d.plugin-v0.1.0.linux-amd64 

```

We should also print that the installer can be run with `--disable-go` to let netdata be installed without go.d.plugin

##### Component Name

installer

##### Steps To Reproduce

Run netdata installer without internet access.

##### Expected behavior

It should fail (like it does), but let the user know there is an option to install netdata without go.d.plugin

**Comment by teraflik** [comment]
What will be the consequences of installing netdata without the aforementioned go.d.plugin, as opposed to a complete netdata installation?

**Comment by cakrit** [comment]
> What will be the consequences of installing netdata without the aforementioned go.d.plugin, as opposed to a complete netdata installation?

You won't have the collector modules listed under https://docs.netdata.cloud/collectors/go.d.plugin/

**Comment by cakrit** [comment]
This was closed in #5460.
